src/web/extension.ts

```diff
@@ -4,9 +4,17 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as vscode from "vscode";
+import TelemetryReporter from "@vscode/extension-telemetry";
+import { AI_KEY } from '../client/constants';
+import { randomUUID } from "crypto";
+
+let _telemetry: TelemetryReporter;
 
 export function activate(context: vscode.ExtensionContext): void {
     console.log("Activated web extension!"); // sample code for testing the webExtension
+    // setup telemetry
+    _telemetry = new TelemetryReporter(context.extension.id, context.extension.packageJSON.version, AI_KEY);
+    _telemetry.sendTelemetryEvent("Start", {'web.userId': randomUUID()});
 
     context.subscriptions.push(
         vscode.commands.registerCommand(
```

Comment:
add a call to `_telemetry.sendTelemetryEvent('StartCommand', {'commandId': '<id here>'});` like is done in the main extension. even if it doesn't do anything yet.